docs/scripts/generate-stories.py

```diff
@@ -63,6 +63,12 @@ def main() -> None:
         logger.info(f'Generating rst files from {area}')
 
         with open(f"{area.lstrip('/')}.rst", 'w') as doc:
+            # Metadata -
+            # https://www.sphinx-doc.org/en/master/usage/restructuredtext/field-lists.html
+            doc.write(
+                f':github_url: https://github.com/teemtee/tmt/blob/main{area}'
```

Comment:
1. The URL is invalid for forks and every branch beyond `main`
2. Some areas are directories, some are fmf files. It's not important to fmf, which can handle both correctly, but here we would need to make a distinction. `https://github.com/teemtee/tmt/blob/main/stories/install` does not exist, `https://github.com/teemtee/tmt/blob/main/stories/install.fmf` does.

Every story object should have `web_link()` method, which returns a link to the corresponding fmf file, that would be perfect, but that's for "edit on github" per section rather than per page. My initial idea would be to collect all web links for the `area` stories, and find their common, shared prefix and use that one. I doesn't like it, but right now I don't have a better one.

Reply:
The URL points to `main`, but it should be valid as long as file path in repo doesn't change. Does it makes sense to propose edits to branches?

For `.fmf` vs directory, the generate script knows which template to invoke, and it can set the URL correctly. What is needed, is the list of this generated entries to check, and generators that produce them. To make sure nothing falls apart.